attractions/autocomplete/autocomplete-option.svelte

```diff
@@ -1,8 +1,19 @@
 <script>
+  /**
+   * @typedef {{name: string; details: string}} Option
+   * @event {{ nativeEvent: MouseEvent }} click
+   */
   import { createEventDispatcher } from 'svelte';
   import escapeRegExp from '../utils/escape-regexp.js';
 
+  /**
+   * @type {Option}
```

Comment:
Why no docstring here? It is pretty self-explanatory, but I imagine it would be confusing to users to leave this without a docstring

Reply:
Maybe we could use something like `The option data to render.`

Reply:
Honestly, just because I didn't know what to write, and it seemed self-explanatory.